src/uct/ib/base/ib_iface.c

```diff
@@ -529,7 +533,12 @@ void uct_ib_iface_fill_ah_attr_from_gid_lid(uct_ib_iface_t *iface, uint16_t lid,
     ah_attr->grh.traffic_class = iface->config.traffic_class;
 
     if (uct_ib_iface_is_roce(iface)) {
-        udp_sport               = iface->config.roce_path_factor * path_index;
+         udp_sport             = iface->config.roce_path_factor * path_index;
+        if (iface->config.en_random_factor) {
+            assert(iface->config.roce_path_factor <= UCT_IB_ROCE_MAX_PATH_FACTOR);
```

Comment:
ucs_assert

Reply:
@yosefe Already applied this comment:
https://github.com/yosefe/ucx/pull/127#discussion_r589420139